bin/tictactoe

```diff
@@ -1,3 +1,23 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+puts "Welcome to Tic-Tac-Toe"
+puts "Please enter the number of players. 1, 2, or 0 for AI"
+input = gets.strip
+if input == "1"
+  game = Game.new(Players::Human.new("X"), Players::Computer.new("O"), Board.new)
+  game.board.display
+  game.play
+elsif input == "2"
+  game = Game.new(Players::Human.new("X"), Players::Human.new("O"), Board.new)
+  game.board.display
+  game.play
+elsif input == "0"
+  game = Game.new(Players::Computer.new("X"), Players::Computer.new("O"), Board.new)
+  game.board.display
+  game.play
+else
+  puts "Sorry, please enter the number of players. 1, 2, or 0 for AI"
+  input = gets.strip
+end
```

Comment:
Prefer to encapsulate CLI logic into a class. For example, the only code in your bin executable should be:

```ruby
#!/usr/bin/env ruby
       
require_relative '../config/environment'

TicTacToeCLI.new.call
```

And then you'd have a file: `lib/tic_tac_toe_cli.rb` with:

```ruby
class TicTacToeCLI
  def initialize
  end

  def call
   # Your game logic
  end
end
```

The code in your bin would generally just go in `call`.